panel/widgets/chatbox.py

```diff
@@ -108,7 +108,6 @@ def __init__(
             align="center",
             margin=8,
             styles=bubble_styles,
-            width_policy="min",
```

Comment:
The width_policies didn't do anything so I'm removing them for simplicity and maintainability.